### ISSUE TITLE
missing 1 required positional argument

### ISSUE THREAD

**Issue by jugi92** [hukkelas/DeepPrivacy#8]
https://github.com/hukkelas/DeepPrivacy/blob/31240b83d68ae235876ffd9d8ab9dff1c148925d/deep_privacy/inference/anonymizer.py#L105

This should be 
anonymized_frames = self.anonymize_images(frames, im_bboxes=im_bboxes)
as im_boxes is a positional argument.
Otherwises this method throws an error when it is called with_keypoints = False which is the default

**Comment by hukkelas** [comment]
Thanks for pointing it out.
I would like to merge your pull request, but first some comments:

- empty `__init__.py` files should not be required with python 3.6 (which is the minimum requirement for this code base)
- The setup.py file seems incomplete and would prefet not to commit this, at this time.

**Comment by jugi92** [comment]
yes I only would ask to include the commit with the positional file:
https://github.com/jugi92/DeepPrivacy/commit/4f6463ccf06d93b664916962dba29290c5ed35b6
I am trying to make it able to install this with "pip install git+" to bring it packaged to a machine with GPU. Thus the additional commits, if I have that, I would create another pull request.

**Comment by hukkelas** [comment]
Thanks, I fixed the pointed out error in [this commit](https://github.com/hukkelas/DeepPrivacy/commit/1e997b7936f3a2937a77cbe48638054e1b5a78af)
